load-test-framework/src/main/java/com/google/pubsub/flic/controllers/GCEController.java

```diff
@@ -431,4 +449,28 @@ private InstanceTemplate defaultInstanceTemplate(String type) {
             .setServiceAccounts(Collections.singletonList(new ServiceAccount().setScopes(
                 Collections.singletonList("https://www.googleapis.com/auth/cloud-platform")))));
   }
+
+  /**
```

Comment:
I don't like this, again this scales O(n) with new types. Please just expose the map. Now I'm starting to feel like you should really just expose the original types map passed in, and do processing on it in the Sheets file.